install/overlays/kubernetes/README.md

```diff
@@ -0,0 +1,50 @@
+# Kubernetes overlay
+
+## Pre-requise
+
+This is an overlay intended for Minkube with the following configuration:
+* Cluster-admin privileges are required
+* Namespace is `default`
+* Operator id is `camel-k`
+* An available registry
+
+## Usage
+
+The following env variable are expected
+
+
+To run from local folder :
+```sh
+kubectl kustomize kustomize/overlays/kubernetes | kubectl create -f -
+```
+
+To run from remote github repository:
+```sh
+kubectl kustomize https://github.com/apache/camel-k/kustomize/overlays/kubernetes | kubectl create -f -
+```
+
+NOTE: to use a different branch add the parameter "ref" to the github repository URL.
+
+
+### Minikube
+
+You can easilly configure minikube with the registry addon.
+
+First get the internal registry service IP from minikube :
+
+```sh
+export KAMEL_REGISTRY_ADDRESS="$(kubectl get service --selector "kubernetes.io/minikube-addons"="registry" --namespace kube-system -o=jsonpath='{.items[0].spec.clusterIP}')"
```

Comment:
For minikube we may not required this any longer. There was some development in the past to enable KEP-1755: https://github.com/apache/camel-k/pull/2696 which is, being able to detect any local registry. It would be nice to try without it and, if it works, remove the registry setting part.

Reply:
Thanks, I will check this. This explains the "placeholder" namespace value I saw in the files.